### PR TITLE
[WIP] Updating OneToMany entities with simple ids

### PR DESCRIPTION
**Pull request by jpetitcolas**

### CODE REVIEW

**Comment by jpetitcolas** [comment]
Using the [rest-demo-app](https://github.com/stanlemon/rest-demo-app/), I can't figure out how to add a comment on a post. Sending below POST request to `/api/comment`:

``` json
{
    "body": "test",
    "created_at": "Feb 3, 2014",
    "post_id": 3
}
```

Gives me the following error: 

``` js
{
    "code": 500,
    "message": "An exception occurred while executing 'INSERT INTO Comment (body, created_at, post_id) VALUES (?, ?, ?)' with params [\"test\", \"2014-02-03\", null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: Comment.post_id"
}
```
I tried with several variants, including the variant suggested on the [NgAdminGeneratorBundle](https://github.com/marmelab/NgAdminGeneratorBundle/pull/27#issuecomment-76535188):

``` js
{
    "body": "test",
    "created_at": "Feb 3, 2014",
    "post": 3
}
```
But same results. Am I missing something?

This PR aims to check the issue, or adds a test if error is mine.

File: Tests/Controller/ResourceControllerTest.php [hunk 1]
```diff
@@ -1034,6 +1032,36 @@ public function testPostWithIdForObject()
...
+
+        $response = $this->controller->postAction($request, 'person');
+        $this->assertTrue($response->isSuccessful());
+
+        $person = $this->em->getRepository('Lemon\RestBundle\Tests\Fixtures\Person')->findOneBy(array(
+            'name' => 'Stan Lemon'
+        ));
+
+        $this->assertNotNull($person);
+        $this->assertEquals(['Mustang'], array_map(function($car) {
```
Comment (stanlemon) [inline Tests/Controller/ResourceControllerTest.php]: For the time being I am still targeting support for php 5.3 so this will need to be changed.

**Comment by stanlemon** [comment]
This seems like a legit issue, and your assumption in the first example of being able to post ```post: 3``` should be correct. I am traveling for work next week and probably won't get to circle around to fixing this until after I get back.  It's a top priority for me though.

**Comment by jpetitcolas** [comment]
Thanks. No problem. I'll continue to fix other relationships issue on the generator bundle meanwhile.

**Pull request merged**